examples/action-listener/counter/package.json

```diff
@@ -12,7 +12,7 @@
     "react-dom": "^18.1.0",
     "react-redux": "^8.0.2",
     "react-scripts": "5.0.1",
-    "typescript": "~4.2.4"
+    "typescript": "~4.3.4"
```

Comment:
typescript-eslint@v6 increases the minimum TS range to 4.3.

Reply:
Ah, it looks like the Codesandbox environment is running on a version of Node too old for nullish coalescing assignment (`??=`)

```plaintext
+ node -v (in /tmp/4c6c9715)
v14.21.1
```

```plaintext
/tmp/4c6c9715/node_modules/@typescript-eslint/eslint-plugin/dist/util/getStringLength.js:16
    splitter ??= new grapheme_splitter_1.default();
             ^^^

SyntaxError: Unexpected token '??='
    at new Script (vm.js:102:7)
```

Per http://kangax.github.io/compat-table/es2016plus/ > **2021 features** > [Logical Assignment](https://github.com/tc39/proposal-logical-assignment/) > _??= basic support_, Node 16 up is needed.